source/_docs/clone-site.md

```diff
@@ -43,7 +43,7 @@ If your database and code compressed are less than 256MB you can exclude the fil
 
 1. Use [Terminus](/docs/terminus) and the `--tar-options` flag:
 
-        terminus drush <site>.<env> ard --tar-options="--exclude=code/sites/default/files" --destination=code/sites/default/files/<RANDOM_HASH>.tgz
+        terminus drush <site>.<env> ard --tar-options="--exclude=code/sites/default/files" --destination=sites/default/files/<RANDOM_HASH>.tgz
 

```

Comment:
@ttrowell this didn't work for me. 
```
Archive saved to                                                            [ok]
/srv/bindings/97769da8f5994eab931429671f45d314/sites/default/files/<RANDOM_HASH>.tgz
```
The suggested URL (e.g. `https://env-site-name.pantheonsite.io/sites/default/files/<RANDOM_HASH>.tgz`) serves a 404

Running destination with `code/` in the path serves the archive on the suggested URL as expected.